server/images.go

```diff
@@ -257,11 +243,14 @@ func CreateModel(ctx context.Context, name, modelFileDir string, commands []pars
 	config := ConfigV2{
 		OS:           "linux",
 		Architecture: "amd64",
+		RootFS: RootFS{
+			Type: "layers",
```

Comment:
Any side-effects to this, or is it just aesthetic?

Reply:
Purely aesthetic. This was being set in `SaveLayers` before unconditionally